extensions/wikia/WikiaWhiteList/WikiaExternalImageList.php

```diff
@@ -53,9 +53,9 @@ function wfExtImagesWhiteListSetup() {
 }
 
 function wfExtImagesWhiteListParse($text) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

[project-id]: wikia:app